server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/mapper/OdcRuntimeDelegateMapper.java

```diff
@@ -72,8 +71,7 @@ public Class<? extends BaseRuntimeFlowableDelegate<?>> map(@NonNull TaskType tas
             case EXPORT_RESULT_SET:
                 return ResultSetExportFlowableTask.class;
             case PRE_CHECK:
-                return enableTaskFramework ? PreCheckRuntimeFlowableTaskCopied.class
-                        : PreCheckRuntimeFlowableTask.class;
+                return PreCheckRuntimeFlowableTask.class;
```

Comment:
The mapper of `ASYNC` task should also specify to `DatabaseChangeRuntimeFlowableTask.class`?

Reply:
yes, new DatabaseChangeRuntimeFlowableTask be closed too when dlm is ready